backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java

```diff
@@ -32,13 +32,13 @@ public OrganizationResolver(OrganizationService os) {
         });
   }
 
-  public List<ApiOrganization> getOrganizations() {
+  public List<ApiOrganization> getOrganizations(Boolean identityVerified) {
```

Comment:
I feel like some documentation of why this is an optional argument might be good here? Confused me initially.